README.md

```diff
@@ -441,20 +441,54 @@ any integration problems with them.
 
 # Advice to researchers
 
-## Git
+## Difficulties in mining git
 
 You should read this paper:
 - Bird, Christian, et al. "The promises and perils of mining git." Mining Software Repositories, 2009. MSR'09. 6th IEEE International Working Conference on. IEEE, 2009. [Link](http://cs.queensu.ca/~ahmed/home/teaching/CISC880/F10/papers/MiningGit_MSR2009.pdf).
 
-# How do I cite RepoDriller?
+# FAQs
+
+## Why use an MSR framework?
+
+There's no question that Mining Software Repositories (MSR) studies benefit from automation.
+The datasets are too large to analyze manually.
+
+So the choice is whether to use an MSR framework or to write your own scripts.
+An MSR framework offers two benefits:
+- The researcher can focus on their questions and not on the infrastructure.
+- Coding against a framework improves standardization and therefore reproducibility (see Robles, Gregorio. "Replicating MSR: A study of the potential replicability of papers published in the Mining Software Repositories proceedings." Mining Software Repositories (MSR), 2010 7th IEEE Working Conference on. IEEE, 2010.).
+
+## How is RepoDriller different from other MSR frameworks?
+
+RepoDriller is a minimalist's MSR framework, a lightweight tool for flexible analysis.
+- RepoDriller is *lightweight*:
+	1. It's a straightforward Java framework with the APIs you need -- no more, no less.
+	2. You pay for storage and computation when you need to. No significant pre-processing stage, no giant database.
+- RepoDriller is *flexible*:
+	1. Write arbitrary analyses in the popular Java programming language.
+	2. RepoDriller has the right knobs -- tune which commits you visit, how much concurrency you want, etc.
+
+Here's how it compares to some other MSR frameworks and tools:
+- [GHTorrent](http://ghtorrent.org/) lets you query GitHub events.
+	1. You are restricted to querying projects on GitHub.
+	2. You are restricted to the information exposed in a GitHub API.
+- [Boa](http://boa.cs.iastate.edu/) lets you query ASTs on a pre-defined set of repositories.
+	1. You are restricted to the repositories tracked by Boa.
+	2. You must write queries in the Boa language, largely against ASTs.
+	3. If you roll your own Boa cluster, you are restricted to repositories with languages that Boa can import (i.e. parse into ASTs).
+- [Alitheia Core](https://github.com/istlab/Alitheia-Core) is a scalable platform for MSR.
+	1. Alitheia-Core is a heavyweight approach. You pay a lot of up-front costs (configuration, pre-processing, etc.) in exchange for a scalable analysis. If you're doing exploratory research, the overhead may not be worth it.
+	2. Alitheia Core is no longer being maintained. You have to figure out how to set it up yourself. Good luck.
```

Comment:
I'd simply remove this _You have to figure out how to set it up yourself. Good luck._